hack/pre-run-e2e.sh

```diff
@@ -15,13 +15,8 @@
 # limitations under the License.
 
 set -o errexit
-set -o pipefail
-
-if [[ "${HC_ACCESS_KEY}" = "" ]] || [[ "${HC_SECRET_KEY}" = "" ]]; then
-  echo "HC_ACCESS_KEY and HC_SECRET_KEY can not be empty, please set it"
```

Comment:
why remove this?

Reply:
the secret has been created in the specified namespace, so we do not need to create it every time we run the E2E tests.

Reply:
ok